pkg/log/log.go

```diff
@@ -110,48 +111,53 @@ type F = logf.F
 
 // Debug emits a log at DEBUG level but only if an error or fatal happens
 // within 2min of this event
-func Debug(_ context.Context, message string, m ...Marshaler) {
+func Debug(ctx context.Context, message string, m ...Marshaler) {
 	dbgEntries.Append(format(message, "DEBUG", time.Now(), app.Info(), m))
```

Comment:
suggestion: missing `ctx`
```suggestion
	dbgEntries.Append(format(ctx, message, "DEBUG", time.Now(), app.Info(), m))
```